travis/build-all-in-one-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Only build docker container for master branch
```

Comment:
nit: my suggestion was to build & test image always, but not to push it to the Hub unless on master